pkg/operator/ceph/config/monstore.go

```diff
@@ -301,12 +312,45 @@ func (m *MonStore) setAll(clientName string, settings map[string]string) ([]stri
 			return []string{}, errors.Wrapf(err, "failed to parse assimilate output file %s", outFilePath)
 		}
 		// get the section for the client
-		section, err := iniContent.GetSection(clientName)
+		section, err := iniContent.GetSection(who)
 		if err != nil {
-			return []string{}, errors.Wrapf(err, "failed to get section %s", clientName)
+			return []string{}, errors.Wrapf(err, "failed to get section %s", who)
 		}
 		return section.KeyStrings(), nil
 	}
 	logger.Info("successfully applied settings to the mon configuration database")
 	return []string{}, nil
 }
+
+// Alias for cluster CRD ceph config options map
+type CephConfigOptionsMap = map[string]map[string]string
```

Comment:
How about placing `type`s and `var`s at the top of the Go file, instead of mixing with the functions.